STEERING-COMMITTEE.md

```diff
@@ -1,32 +1,115 @@
-2. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
-3. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository.
-4. Control and delegate access to and establish processes regarding other project resources and assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
-5. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
-6. Resolve any dispute from the Technical Oversight Committee.
+The Steering Committee’s responsibilities include:
+
+1. Establishing rules of governance for the Istio project, including creation
+and ratification of bylaws for the Steering Committee and Technical Oversight
+Committee.
+1. Establishing and enforcing principles to guide the Istio project.
+1. Fostering an environment for a healthy and happy community of developers,
+contributors, vendors, and users.
+1. Defining, evolving, and defending a
+[Code of Conduct](CONTRIBUTING.md#code-of-conduct).
+1. Advising the Open Usage Commons on issues relating to the Istio trademark and
+logo, as well as related conformance programs.
+1. Setting marketing and advocacy direction for the project; establishing a
+publishing schedule and vetting content, encouraging and assisting community
+members in creating content for conferences, fostering an ecosystem of vendors.
+1. Controlling and delegating access to, and establishing processes regarding,
+project resources/assets, including but not limited to artifact repositories,
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+
+1. A Company shall mean an entity which employs a member, and all other
+entities that control, are controlled by, or are under common control with that
+entity.
+1. The Steering Committee is structured to allow Companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1. All members should abide by the project Code of Conduct.
+1. There are two types of seats on the Steering Committee: Contribution Seats
+and Community Seats.
+1. Both Contribution and Community Seats will be appointed beginning in
+August 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1. Contribution Seat terms expire on January 31 and Community Seat terms expire
+on July 31. If necessary, a company holding a Contribution Seat may change the
+appointed individual at any time during the term.
```

Comment:
I assume this implies that the company can choose to change this multiple times throughout the year without any approval from steering?

Reply:
In theory, yes.  That would change who issues the vote on their behalf, but not how many votes that Company gets.

Reply:
I have the same reading. Any company that wants to change who holds their contributor seat(s) should just make a PR to update who is the official holder to record the change.